lib/qa/authorities/linked_data/find_term.rb

```diff
@@ -75,14 +73,14 @@ def load_graph(url:)
         def normalize_results
           normalize_start_dt = Time.now.utc
 
-          json = perform_normalization
+          results = perform_normalization
```

Comment:
Results is more accurate name.  It should have changed when support for n3 and ntriples was added.